hub/cli/auth.py

```diff
@@ -10,19 +10,20 @@
 @click.option("--password", "-p", default=None, help="Your Activeloop password")
 def login(username: str, password: str):
     """Log in to Activeloop"""
-    click.echo("Log in using Activeloop credentials.")
+    click.echo("Login to Activeloop Hub using your credentials.")
     click.echo(
-        "If you don't have an account register by using 'hub register' command or by going to "
+        "If you don't have an account, register by using 'activeloop register' command or by going to "
         "https://app.activeloop.ai/register."
```

Comment:
This is a nitpick but we can use the prefix constant we use in `HubBackendClient`. Just in case things change.